testApp/src/Components/MovieDetail.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    import type { Movie } from '../Interfaces/Movie';
+    import { scale } from 'svelte/transition';
+
+    export let movie: Movie;
+    const dispatch = createEventDispatcher();
+
+    function backToGallery() {
+        dispatch('backToGallery');
+    }
+</script>
+
+<div class="movie-detail" on:click={backToGallery} in:scale={{ duration: 500 }} out:scale={{ duration: 500 }}>
```

Comment:
It would probably be nice to be able to cancel it by pressing Escape. The same assistive technology feedback applies here as well, btw.

Reply:
I added the role button to the div that is what you meant right ?

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-ef63d3406a68b222fc856f03cd2218f2a16eb0956f5261a5eb6cc1970f470bca

Reply:
I see it's implemented better in your next PR. Will resolve this one.